### PR TITLE
Export as methods

### PR DESCRIPTION
**Pull request by flying-sheep**
I figured that this works better after all.

With a `export`, the `altair` method needs to be loaded to be able to display a chart. With a `S3method`, it doesn’t.

So an `S3Method`-style export is better for people who use e.g. [import](https://cran.r-project.org/web/packages/import/vignettes/import.html) or `altair$alt$Chart(...)` instead of `library(altair)`

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -3,24 +3,24 @@
 S3method("&",altair.vegalite.v2.api.TopLevelMixin)
 S3method("+",altair.vegalite.v2.api.TopLevelMixin)
 S3method("|",altair.vegalite.v2.api.TopLevelMixin)
+S3method(as_vegaspec,altair.vegalite.v2.api.TopLevelMixin)
```
Comment (flying-sheep) [inline NAMESPACE]: roxygen did this for me, but it seems correct!

File: R/utils-jupyterlab.R [hunk 2]
```diff
@@ -15,10 +15,11 @@ py_mimebundle <- function(obj) {
 #'
 #' @return character
 #' @keywords internal
+#' @importFrom repr repr_text
 #' @export
 #'
 repr_text.altair.vegalite.v2.api.Chart <- function(obj) {
-  py_mimebundle(chart)$`text/plain`
+  py_mimebundle(obj)$`text/plain`
```
Comment (flying-sheep) [inline R/utils-jupyterlab.R]: you also forgot to rename chart→obj here

**Comment by ijlyttle** [comment]
Thanks!

File: R/utils-jupyterlab.R (hunk 2 above)
Comment (ijlyttle) [reply R/utils-jupyterlab.R]: Doh... Thanks!

**Pull request merged**